test/test_parse_psse.jl

```diff
@@ -201,3 +201,79 @@ end
     @test isapprox(get_inverter_extinction_angle_limits(lcc).max, pi / 2)
     @test get_power_mode(lcc)
 end
+
+@testset "PSSE Impedance Correction Table Parsing" begin
+    base_dir = string(dirname(@__FILE__))
+    file_dir = joinpath(base_dir, "test_data", "modified_14bus_system.raw")
+    sys = System(file_dir)
+
+    tr2w_1 = get_component(Transformer2W, sys, "BUS 110-BUS 109-i_1")
+    suppl_attr_tr2w_1 = get_supplemental_attributes(tr2w_1)[1]
+    @test get_table_number(suppl_attr_tr2w_1) == 3
+    @test get_impedance_correction_curve(suppl_attr_tr2w_1).points[1] ==
+          (x = -24.1, y = 1.27)
+    @test get_impedance_correction_curve(suppl_attr_tr2w_1).points[end] ==
+          (x = 24.1, y = 1.27)
+    @test get_transformer_winding(suppl_attr_tr2w_1) == WindingCategory.TR2W_WINDING
+    @test get_transformer_control_mode(suppl_attr_tr2w_1) ==
+          TransformerControlMode.PHASE_SHIFT_ANGLE
+
+    tr2w_2 = get_component(Transformer2W, sys, "BUS 109-BUS 104-i_1")
+    suppl_attr_tr2w_2 = get_supplemental_attributes(tr2w_2)[1]
+    @test get_table_number(suppl_attr_tr2w_2) == 1
+    @test get_impedance_correction_curve(suppl_attr_tr2w_2).points[1] ==
+          (x = 0.88, y = 1.12)
+    @test get_impedance_correction_curve(suppl_attr_tr2w_2).points[end] ==
+          (x = 1.17, y = 0.895)
+    @test get_transformer_winding(suppl_attr_tr2w_2) == WindingCategory.TR2W_WINDING
+    @test get_transformer_control_mode(suppl_attr_tr2w_2) ==
+          TransformerControlMode.TAP_RATIO
+
+    tr3w_1 = get_component(Transformer3W, sys, "BUS 109-BUS 104-BUS 107-i_1")
```

Comment:
Can we test the case where one three-winding transformer has ICT entries for multiple of its windings?

Reply:
Also

- Multiple transformers have the same ICT entry on the same winding
- Multiple transformers have the same ICT entry on different windings